src/JoomlaBrowser.php

```diff
@@ -1164,31 +988,154 @@ public function createCategory($title, $extension = '')
 		}
-
 		$this->amOnPage('administrator/index.php?option=com_categories' . $extension);
-
 		$this->waitForElement(['class' => 'page-title']);
 		$this->checkForPhpNoticesOrWarnings();
-
 		$this->debug('Click new category button');
 		$this->click($this->locator->adminToolbarButtonNew);
-
 		$this->waitForElement(['class' => 'page-title']);
-
 		$this->checkForPhpNoticesOrWarnings();
 		$this->fillField(['id' => 'jform_title'], $title);
-
 		$this->debug('Click new category apply button');
 		$this->click($this->locator->adminToolbarButtonApply);
-
 		$this->debug('see a success message after saving the category');
-
 		$this->see('Category saved', ['id' => 'system-message-container']);
 		$this->checkForPhpNoticesOrWarnings();
 	}
+	/**
+	 * Site Offline
+	 *
+	 * @return  void
+	 *
+	 * @since   3.7.5
+	 */
+	public function setSiteOffline($enable == true)
+	{
+        	$this->debug('I set the site to offline');
+        	$this->doAdministratorLogin();
+        	$this->amOnPage('/administrator/index.php?option=com_config');
+		if($enable == true){
+			$this->click(['id' => 'jform_offline1']);
+			$this->click("Save");
+			$this->see('Configuration saved', ['id' => 'system-message-container']);
+		}else{
+			$this->click(['id' => 'jform_offline0']);
+			$this->click("Save");
+			$this->see('Configuration saved', ['id' => 'system-message-container']);
+		}
+	}
+	/**
+	 * Search Engine Optimization
+	 *
+	 * @param   \AcceptanceTester   $I  Acceptance Tester
+	 *
+	 * @return void
+	 */
+	public function setSiteSearchEngineFriendly($enable == true)
+	{
+        	$this->debug('I set the Search engine optimisation to Yes');
+        	$this->doAdministratorLogin();
+        	$this->amOnPage('/administrator/index.php?option=com_config');
+		if($enable == true){
+			$this->click(['id' => 'jform_sef1']);
+			//Select Option 'After'
+			$this->selectOption(['id' => 'jform_sitename_pagetitles'],'After');
+			$this->click("Save");
+			$this->see('Configuration saved', ['id' => 'system-message-container']);
+		}else{
+			$this->click(['id' => 'jform_sef0']);
+			$this->click("Save");
+			$this->see('Configuration saved', ['id' => 'system-message-container']);
+		}
+	}
+	/**
+     	 * Create Module
+     	 *
+     	 * @param   \AcceptanceTester $I          Acceptance Tester
+     	 * @param   string            $scenario   Scenario
+       	 *
+     	 * @return void
+     	 */
+	public function createModule($moduleType, $moduleName, $category)
+    	{
+		$this->doAdministratorLogin();
+        	$this->amOnPage('/administrator/index.php?option=com_modules');
+        	// New Module
+        	$this->click('New');
+        	// Select Module Type
+        	$this->see('Select a Module Type');
+        	/**
+         	 * Module Types
+         	 * Articles - Archived
+         	 * Articles - Categories
+         	 * Articles - Category
+         	 */
+        	switch ($moduleType)
+        	{
+            	case 'Articles - Archived' :
```

Comment:
I think the styling is all wrong here, also please make sure you only work with branch 4.0 on Joomla Browser, your PR should also be on 4.0, don't send PR for develop branch.